.circleci/config.yml

```diff
@@ -522,6 +525,46 @@ jobs:
       - deploy-to-staging-perf:
           project_name: "all-of-us-rw-perf"
 
+  puppeteer-generate-access-tokens:
+    executor: puppeteer-executor
+    parameters:
+      env_name:
+        description: The target environment for run Puppeteer end-to-end test. Must be one of "test", "staging", "local", "perf".
+        default: "test"
+        type: enum
+        enum: [ "test", "staging", "local", "perf" ]
+    steps:
+      - checkout-code
+      - restore_cache:
+          name: "Restore api gradle wrapper cache"
+          key: v6-gradle-wrapper-{{ .Branch }}-{{ checksum "~/workbench/api/gradle/wrapper/gradle-wrapper.properties" }}
+      - restore_cache:
+          name: "Restore api gradle cache"
+          key: v6-gradle-cache-{{ .Branch }}-{{ checksum "~/workbench/api/build.gradle" }}
+      - restore_cache:
+          name: "Restore e2e yarn cache"
+          keys:
+            - v5-e2e-yarn-{{ .Branch }}-{{ arch }}-{{ checksum "~/workbench/e2e/yarn.lock" }}
+      - run:
+          name: "Set environment variables for e2e test users"
+          working_directory: ~/workbench/e2e
+          command: |
+            cp .env.circleci.<< parameters.env_name >> .env
+      - run:
+          # Needed for env-cmd to function on Circle.
+          command: touch ~/workbench/e2e/.env
```

Comment:
👍🏻 removed.

Reply:
FYI: I'm seeing a lot of `env-cmd: not found` errors after merging this PR.  Do we in fact need this?  Unclear to me why this would help. @calbach @aweng98

Reply:
Here is an apparent fix for the issue based on a different theory https://github.com/all-of-us/workbench/pull/6095